MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -319,7 +313,12 @@ public void FixedUpdate()
 
 		private void SendPhysicsUpdate()
 		{
-			PhysicsBridgePatch physicsPatch = new PhysicsBridgePatch(InstanceId,
+			if (LocalUser == null)
+			{
+				return;
+			}
+
+			PhysicsBridgePatch physicsPatch = new PhysicsBridgePatch(LocalUser.Id,
```

Comment:
Does this fixes the authority problem, to use LocalUser.Id instead of InstanceId>

Reply:
No, but only userID is exposed in SDK for a app dev

---
In reply to: [445051282](https://github.com/microsoft/mixed-reality-extension-unity/pull/199#discussion_r445051282) [](ancestors = 445051282)